Graph Theory/Breadth First Search Algorithm/Java/Graph.java

```diff
@@ -0,0 +1,91 @@
+/* ===== ===== =====
+
+Theory of Programming
+
+Breadth First Search Algorithm
+http://theoryofprogramming.com/2014/12/25/breadth-first-search-algorithm/
+GitHub - https://github.com/VamsiSangam/theoryofprogramming
+Code Contributor - Saurabh Rane (github.com/ranesr)
+
+===== ===== ===== */
+
+import java.util.Iterator;
+import java.util.LinkedList;
+
+public class Graph {
+
+	// No. of vertices
+	private int v;
+
+	// Adjacency Lists
+	private LinkedList<Integer> adj[];
+
+	// Constructor for Graph
+	@SuppressWarnings("unchecked")
+	public Graph(int v) {
+		this.v = v;
+		adj = new LinkedList[v];
+		for (int i=0; i<v; ++i) {
```

Comment:
We really want spaces on either side of every binary operator in the whole file. It improves readability.